Description: Preserves the Files.fileTreeTraverser() method that was removed from the public API in Guava 25.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/guava/src/com/google/common/io/Files.java
+++ b/guava/src/com/google/common/io/Files.java
@@ -933,7 +933,7 @@
    *     {@link #fileTraverser()} instead with the same semantics as this method.
    */
   @Deprecated
-  static TreeTraverser<File> fileTreeTraverser() {
+  public static TreeTraverser<File> fileTreeTraverser() {
     return FILE_TREE_TRAVERSER;
   }
 
